tests/conftest.py

```diff
@@ -16,7 +16,7 @@
 from .emulators import emulator, emulator_rv
 
 TESTDATA_BRANCH = os.getenv("RAVENPY_TESTDATA_BRANCH", "master")
-SKIP_TEST_DATA = os.getenv("RAVENPY_SKIP_TEST_DATA")
+SKIP_TEST_DATA = True  # os.getenv("RAVENPY_SKIP_TEST_DATA")
```

Comment:
Be sure to change this back before merging!

Reply:
Yep !